extensions/amp-dailymotion/0.1/amp-dailymotion.js

```diff
@@ -302,6 +303,14 @@ class AmpDailymotion extends AMP.BaseElement {
     const implicitParams = getDataParamsFromAttributes(this.element);
     iframeSrc = addParamsToUrl(iframeSrc, implicitParams);
 
+    // In order to support autoplay the video needs to be muted on load so we
```

Comment:
What sends the unmute event?

Reply:
I am not actually sure, it doesn't seem to be based on user interaction but when autoplay is enabled I can see it come through as the first event.

Reply:
Does it come from our code or the dailymotion iframe?

Reply:
Ah, I understand:  It's from the dailymotion iframe.

Reply:
Cool cool. Working around the iframe SGTM